rero_ils/modules/documents/dojson/contrib/marc21tojson/model.py

```diff
@@ -293,8 +293,10 @@ def marc21_to_publishers_publicationDate(self, key, value):
 
     if publisher:
         publishers.append(publisher)
-
-    return publishers
+    if not publishers:
+        return None
+    else:
```

Comment:
Optional: The `else` here is not necessary, you can safely delete line #298
`setup.py` works fine, all is good